src/test/resources/test_4.csv

```diff
@@ -0,0 +1 @@
+sdasdasdasda
```

Comment:
missing empty line in the end of the file, same with other .csv files

Reply:
could you explain, why do we need this empty lines everywhere?

Reply:
https://stackoverflow.com/questions/4700312/empty-new-line-at-the-end-of-the-java-source-files